modules/example/src/main/scala/zio/telemetry/example/http/Status.scala

```diff
@@ -3,9 +3,13 @@ package zio.telemetry.example.http
 import io.circe._
 import io.circe.generic.semiauto._
 
-final case class Status(name: String, version: String, status: String)
+final case class Status private (name: String, status: String)
```

Comment:
You can leave it public. If you want it to be properly private, more ceremony is needed (disabling copy), but there's no need for that.